src/WorldStorage/WSSAnvil.h

```diff
@@ -150,11 +150,14 @@ class cWSSAnvil :
 	This function extracts the text from either version. */
 	AString DecodeSignLine(const AString & a_Line);
 
-	/** Returns true iff the "id" child tag inside the specified tag equals (case-sensitive) any of the specified expected types.
+	/** Returns true if the "id" child tag inside the specified tag equals (case-sensitive) any of the specified expected types.
```

Comment:
Actually `iff` is the correct terminus technicus, being short for "if and only if". But whatever